float32 A
float32 B
float32 X
float32 Y
float32 LB
float32 RB
float32 back
float32 start
float32 power
float32 stick_button_left
float32 stick_button_right
float32 dpad_horizontal
float32 dpad_vertical
float32 LT
float32 RT
float32 LS_horizontal
float32 LS_vertical
float32 RS_horizontal
float32 RS_vertical
